include/regs.h

```diff
@@ -199,4 +199,150 @@
 #define R_EN_GOROIWA_SPEED                       mREG(12)
 #define R_NAVI_MSG_REGION_ALPHA                  nREG(87)
 
+
+#define R_HREG_MODE  HREG(80) // see `HRegMode` for mode options
+
+typedef enum {
+    /*  7 */ HREG_MODE_PRINT_HILITE_INFO = 6, // print hilite information
```

Comment:
```suggestion
    /*  6 */ HREG_MODE_PRINT_HILITE_INFO = 6, // print hilite information
```
whoopsie
also nice to see I did enough bullying to make this change happen :sunglasses:

Reply:
i still dont really agree with it, but yeah lol
I dont think either of these were really a part of this system, but we will never have a way of knowing